src/frontend-pwa/src/components/SearchBar/searchbar.styles.ts

```diff
@@ -0,0 +1,46 @@
+import styled from '@emotion/styled';
+import typography from '../../typography';
+
+export const WrapperDiv = styled.div`
+    max-width: 1000px;
+    margin: auto; 
+    width: 320px; 
+    height: 80%;
+    position: relative;
+    display: inline-block;
+`;
+
+export const StyledInput = styled.input`
+  ${typography.toString()}
+  max-width: 800px;
+  width: 80%;
+  padding: 2px 50px 2px 5px;
+  border-radius: 10px;
+  font-size: 15px;
```

Comment:
Don't mess around too much with `font-size` and other globally set properties. Typically, UI/UX will provide a very strict typographical guide—body text is usually `font-size: 16pt`, as an example—there are very few times where you will be overriding something like this.

Reply:
duly noted! was trying to get a "stub" up, and perhaps ran roughshod over the guide in doing do. will refine going forward.